pom.xml

```diff
@@ -336,6 +336,11 @@
 				<artifactId>hapi-base</artifactId>
 				<version>2.1</version>
 			</dependency>
+			<dependency>
+				<groupId>org.codehaus.jackson</groupId>
```

Comment:
Is this a formatting change?

Reply:
NO this wasn't necessary, am going to work on it